### ISSUE TITLE
Update Byte Stream Handling

### ISSUE THREAD

**Issue by elswenson** [echistyakov/GauchoGrub2#10]
@echistyakov look into the removal of ByteArrayBuffer and replace it as necessary. It no longer seems to be a relevant class? 

like here in WebUtils: 

```
private byte[] readByteStream(InputStream stream) throws IOException {
        BufferedInputStream reader = new BufferedInputStream(stream);
        ByteArrayBuffer response = new ByteArrayBuffer(100000);
        byte[] buffer = new byte[1024];
        int length;
        while ((length = reader.read(buffer)) != -1) {
            response.append(buffer, 0, length);
        }
        reader.close();
        return response.toByteArray();
    }
```

**Comment by echistyakov** [comment]
@elswenson Pull request ready: #14 
The idea about Google Commons IO taken from here: https://github.com/Cielo24/cielo24-java/blob/master/cielo24/src/cielo24/WebUtils.java

**Comment by elswenson** [comment]
Merged #14 to handle issue.
